cron/worker/main.go

```diff
@@ -217,6 +217,11 @@ func main() {
 	}()
 
 	checksToRun := checks.AllChecks
+	// Remove checks which require lot of GitHub API token.
```

Comment:
Add TODO: and say Remove checks temporarily

Reply:
Done.